File: app/src/main/java/com/stevenfrew/beatprompter/event/BeatEvent.java
package com.stevenfrew.beatprompter.event;

public class BeatEvent extends BaseEvent
{
    public double mBPM;
    public int mBPB;
    public int mBPL;
    public int mBeat;
    public boolean mClick;
    public int mWillScrollOnBeat;

    public BeatEvent(long eventTime,double bpm,int bpb,int bpl, int beat,boolean click,int willScrollOnBeat)
    {
        super(eventTime);
        mBPL=bpl;
        mPrevBeatEvent=this;
        mBPM=bpm;
        mBPB=bpb;
        mBeat=beat;
        mClick=click;
        mWillScrollOnBeat=willScrollOnBeat;
    }
}

File: app/src/main/java/com/stevenfrew/beatprompter/event/BeatEvent.kt
package com.stevenfrew.beatprompter.event

class BeatEvent(eventTime: Long, var mBPM: Double, var mBPB: Int, var mBPL: Int, var mBeat: Int, var mClick: Boolean, var mWillScrollOnBeat: Int) : BaseEvent(eventTime) {
    init {
        mPrevBeatEvent = this
    }
}
